cmd/crates/soroban-spec-typescript/ts-tests/initialize.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# read .env file, but prefer explicitly set environment variables
+IFS=$'\n'
+for l in $(cat .env); do
+    IFS='=' read -ra VARVAL <<< "$l"
+    # If variable with such name already exists, preserves its value
+    eval "export ${VARVAL[0]}=\${${VARVAL[0]}:-${VARVAL[1]}}"
+done
+unset IFS
+
+echo Network
+echo "  RPC:        $SOROBAN_RPC_URL"
+echo "  Passphrase: \"$SOROBAN_NETWORK_PASSPHRASE\""
+
+# Print command before executing, from https://stackoverflow.com/a/23342259/249801
+# Discussion: https://github.com/stellar/soroban-tools/pull/1034#pullrequestreview-1690667116
+exe() { echo"${@/eval/}" ; "$@" ; }
+
+function fund_all() {
+  exe eval "./soroban config identity fund"
+  exe eval "./soroban config identity generate alice"
```

Comment:
If the network is set via ENV vars than this should pre-fund the account. Do you see any warnings in the output of the next line?

Reply:
Yes, there's a warning that the account already exists.